jasmine/src/inverted-index.js

```diff
@@ -0,0 +1,149 @@
+class Index {
+
+    // Method to create an inverted-index of the documents
+    createIndex(filePath) {
+
+        // Here, self refers to the Index object
+        var self = this;
+
+        // Initialise Object to store the inverted-index
+        this.indexObj = {};
+
+        /* Fetch the document 
+         * Resolve the promise and store the data in a variable, 'finallySomeData'
+         */
+        return fetch(filePath).then(function(response) {
+            return response.json();
+
+        })
+
```

Comment:
Check out this >>> https://github.com/airbnb/javascript#whitespace--padded-blocks